### PR TITLE
Send git aliases to server during setup

### PR DESCRIPTION
**Pull request by tapishr**
Please check the action items covered in the PR - 

- [x] Build is running
- [ ] Eventing is functional and tested
- [ ] Unit or integration tests added and running
- [ ] Manual QA

## Summary by CodeRabbit

- **New Features**
	- Enhanced repository setup with automatic alias handling for both Bitbucket and GitHub integrations.
	- Introduced the ability to retrieve and send unique email aliases from local Git repositories.

- **Improvements**
	- Improved logging for GitHub pull request processing and setup information sharing.

- **Refactor**
	- Updated utility functions for enhanced alias management and communication with external services.

### CODE REVIEW

File: vibi-dpu/src/core/bitbucket/setup.rs [hunk 1]
```diff
@@ -40,6 +42,12 @@ pub async fn handle_install_bitbucket(installation_code: &str) {
             let token_copy = access_token.clone();
             let mut repo_copy = repo.clone();
             clone_git_repo(&mut repo_copy, &token_copy, &repo_provider).await;
+            let aliases_opt = get_git_aliases(&repo_copy);
+            if aliases_opt.is_none() {
+                continue;
```
Comment (MuskanPaliwal) [inline vibi-dpu/src/core/bitbucket/setup.rs:47]: You might want to add log message here.

File: vibi-dpu/src/utils/gitops.rs [hunk 2]
```diff
@@ -610,3 +610,41 @@ pub async fn clone_git_repo(repo: &mut Repository, access_token: &str, repo_prov
     repo.set_local_dir(&directory);
     save_repo_to_db(repo);
 }
+
+pub fn get_git_aliases(repo: &Repository) -> Option<Vec<String>> {
+	log::debug!("[get_git_aliases] repo = {:?}", &repo);
+	let local_dir_opt = repo.local_dir().to_owned();
+	if local_dir_opt.is_none() {
+		return None;
```
Comment (MuskanPaliwal) [inline vibi-dpu/src/utils/gitops.rs:618]: add log message here

**Review by MuskanPaliwal — CHANGES_REQUESTED** [review]
good to go from my end

File: vibi-dpu/src/core/bitbucket/setup.rs (hunk 1 above)
Comment (tapishr) [reply vibi-dpu/src/core/bitbucket/setup.rs:48]: done

File: vibi-dpu/src/utils/gitops.rs (hunk 2 above)
Comment (tapishr) [reply vibi-dpu/src/utils/gitops.rs:619]: done

**Review by MuskanPaliwal — APPROVED** [review]

**Pull request merged**